<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WargearProfileEditDialog</class>
 <widget class="QDialog" name="WargearProfileEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="wargearLabel">
       <property name="text">
        <string>Wargear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="wargearEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rangeLabel">
       <property name="text">
        <string>Range*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="rangeEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="strengthLabel">
       <property name="text">
        <string>Strength*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="strengthEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="apLabel">
       <property name="text">
        <string>AP*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="apEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="rulesGroup">
     <property name="title">
      <string>Rules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="ruleList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../../libwar/resources/images.qrc">
            <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../../libwar/resources/images.qrc">
            <normaloff>:/icons/document-edit.png</normaloff>:/icons/document-edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../../libwar/resources/images.qrc">
            <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="saveBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../libwar/resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
